package.json

```diff
@@ -24,10 +24,17 @@
     "webpack-cli": "^4.10.0"
   },
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
     "css-loader": "^6.7.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
     "html-webpack-plugin": "^5.5.0",
     "style-loader": "^3.3.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
     "webpack-dev-server": "^4.9.3"
```

Comment:
- [ ] All webpack related packages should be added to the devDependencies list.

Reply:
@mikethreels this is not the main requirement, you could say this in optional change, and also you did not explain why its important to add them under devDependencies, and you did not explain the steps on how to move these packages into the devDependencies list.????